resources/tracing/templates/kyma-additions/authorization-policy.yaml

```
@@ -8,16 +8,18 @@ metadata:
 spec:
   action: ALLOW
   rules:
-  - from:
-    - source:
-        principals:
-        - cluster.local/ns/{{ .Release.Namespace }}/sa/{{ include "jaeger-operator.fullname" . }}-auth-proxy
-        - cluster.local/ns/{{ .Release.Namespace }}/sa/kiali
-        - cluster.local/ns/{{ .Release.Namespace }}/sa/monitoring-grafana
-    to:
+  - to:
       - operation:
           ports: # Jaeger query port, limit to auth-proxy, Kiali and Grafana
           - "16686"
+{{ if .Values.authProxy.enabled }}
+    from:
+    - source:
+        principals:
+          - cluster.local/ns/{{ .Release.Namespace }}/sa/{{ include "jaeger-operator.fullname" . }}-auth-proxy
+          - cluster.local/ns/{{ .Release.Namespace }}/sa/kiali
+          - cluster.local/ns/{{ .Release.Namespace }}/sa/monitoring-grafana
+{{ end }}
```

Shouldn't we just limit the scope of the if to the auth proxy principal
```suggestion
    from:
    - source:
        principals:
{{ if .Values.authProxy.enabled }} 
          - cluster.local/ns/{{ .Release.Namespace }}/sa/{{ include "jaeger-operator.fullname" . }}-auth-proxy
{{ end }}  
          - cluster.local/ns/{{ .Release.Namespace }}/sa/kiali
          - cluster.local/ns/{{ .Release.Namespace }}/sa/monitoring-grafana
```